demo/lib/demo/repo.ex

```diff
@@ -2,12 +2,12 @@ defmodule Demo.Repo do
   use Ecto.Repo,
```

Comment:
If we combine the two: Should we use `AshPostgres.Repo` then?

> This repo is a thin wrapper around an `Ecto.Repo`.

https://hexdocs.pm/ash_postgres/AshPostgres.Repo.html

Then we also would not have to implement `min_pg_version`.

Reply:
yes, then we should switch!

Reply:
done

Reply:
@pehbehbeh you should implement `min_pg_version/0`. Otherwise, there is a warning. See https://github.com/ash-project/ash_postgres/blob/v2.4.11/lib/repo/before_compile.ex#L8 and https://github.com/naymspace/backpex/actions/runs/11704207105/job/32596160978